packages/plugins/plugin-chess/src/components/index.ts

```diff
@@ -5,5 +5,4 @@
 import React from 'react';
 
 // Lazily load components for content surfaces.
-export const ChessMain = React.lazy(() => import('./ChessMain'));
-export const ChessArticle = React.lazy(() => import('./ChessArticle'));
+export const ChessWrapper = React.lazy(() => import('./ChessWrapper'));
```

Comment:
ChessContainer? if aligning with other plugins

Reply:
ah yes, that was the intention. thanks.